packages/fast-colors/src/colorlib/colorConverters.ts

```diff
@@ -0,0 +1,385 @@
+import { ColorHSL } from "./colorHSL";
+import { ColorHSV } from "./colorHSV";
+import { ColorLAB } from "./colorLAB";
+import { ColorLCH } from "./colorLCH";
+import { ColorRGBA64 } from "./colorRGBA64";
+import { ColorXYZ } from "./colorXYZ";
+import { degreesToRadians, radiansToDegrees } from "./mathUtils";
+
+// All hue values are in degrees rather than radians or normalized
+// All conversions use the D65 2 degree white point for XYZ
+// Info on conversions and constants used can be found in the following:
+// https://en.wikipedia.org/wiki/CIELAB_color_space
+// https://en.wikipedia.org/wiki/Illuminant_D65
+// https://ninedegreesbelow.com/photography/xyz-rgb.html
+// http://user.engineering.uiowa.edu/~aip/Misc/ColorFAQ.html
+// https://web.stanford.edu/~sujason/ColorBalancing/adaptation.html
+// http://brucelindbloom.com/index.html
+
+// The alpha channel of the input is ignored
+export function rgbToLuminance(rgb: ColorRGBA64): number {
+    function luminanceHelper(i: number): number {
+        if (i <= 0.03928) {
+            return i / 12.92;
+        } else {
+            return Math.pow((i + 0.055) / 1.055, 2.4);

```

Comment:
nit: same comment as earlier, all else values with no qualifiers and a return can just return without the else wrapper.